### PR TITLE
Add sixs and additional warp failure error handlers

### PR DESCRIPTION
**Pull request by dhill-usgs**
We caught a couple new errors over the weekend, added them into the auto error resolver.  Please view changes and comment.  (also had to comment out the postgres db config from this branch since we not quite ready for that yet)

### CODE REVIEW

File: web/espa_web/settings.py [hunk 1]
```diff
@@ -112,14 +112,14 @@
...
         'PORT': config.get('config', 'dbport'),     # Set to empty string for default. Not used with sqlite3.
     },
-    'postgres': {
-        'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': config.get('config', 'post-db'),
-        'USER': config.get('config', 'post-user'),
-        'PASSWORD': config.get('config', 'post-pass'),
-        'HOST': config.get('config', 'post-host'),
-        'PORT': config.get('config', 'post-port')
-    }
```
Comment (rdilley-usgs) [inline web/espa_web/settings.py]: This looks good

File: web/ordering/errors.py [hunk 2]
```diff
@@ -169,10 +170,10 @@ def night_scene(self, error_message):
         return self.__find_error(error_message, keys, status, reason)
 
     def missing_aux_data(self, error_message):
-        '''Could not run do to aux data no available yet'''
+        '''Could not run due to aux data no available yet'''
```
Comment (rdilley-usgs) [inline web/ordering/errors.py]: Ok

File: web/ordering/errors.py [hunk 3]
```diff
@@ -169,10 +170,10 @@ def night_scene(self, error_message):
...
 
         keys = ['Verify the missing auxillary data products',
-                'Could not find auxnm data file:']
+                'Warning: main : Could not find auxnm data file']
```
Comment (rdilley-usgs) [inline web/ordering/errors.py]: Ok

File: web/ordering/errors.py [hunk 4]
```diff
@@ -222,7 +223,9 @@ def lta_soap_errors(self, error_message):
         return self.__find_error(error_message, keys, status, reason, extras)
 
     def warp_errors(self, error_message):
-        keys = ['GDAL Warp failed to transform']
+        keys = ['GDAL Warp failed to transform',
+                'ERROR 1: Too many points',
+                'unable to compute output bounds']
```
Comment (rdilley-usgs) [inline web/ordering/errors.py]: Ok

File: web/ordering/errors.py [hunk 5]
```diff
@@ -28,6 +28,7 @@ def __init__(self, name=None):
         self.conditions.append(self.no_such_file_or_directory)
         self.conditions.append(self.oli_no_sr)
         self.conditions.append(self.only_only_no_thermal)
+        self.conditions.append(self.sixs_errors)
```
Comment (rdilley-usgs) [inline web/ordering/errors.py]: Should I see some other change for sixs_errors elsewhere in the code?

File: web/espa_web/settings.py [hunk 6]
```diff
@@ -112,14 +112,14 @@
...
+    #'postgres': {
+    #    'ENGINE': 'django.db.backends.postgresql_psycopg2',
+    #    'NAME': config.get('config', 'post-db'),
+    #    'USER': config.get('config', 'post-user'),
+    #    'PASSWORD': config.get('config', 'post-pass'),
+    #    'HOST': config.get('config', 'post-host'),
+    #    'PORT': config.get('config', 'post-port')
+    #}
```
Comment (rdilley-usgs) [inline web/espa_web/settings.py]: Ok

File: web/ordering/errors.py (hunk 5 above)
Comment (dhill-usgs) [inline web/ordering/errors.py]: no, the error handler was already there (the function sixs_errors was
defined) but wasn't included in the conditions to check.

On Tue, Jul 28, 2015 at 2:32 PM, Ronald D. Dilley Jr. <
notifications@github.com> wrote:

> In web/ordering/errors.py
> <https://github.com/USGS-EROS/espa/pull/319#discussion_r35690135>:
>
> > @@ -28,6 +28,7 @@ def __init__(self, name=None):
> >          self.conditions.append(self.no_such_file_or_directory)
> >          self.conditions.append(self.oli_no_sr)
> >          self.conditions.append(self.only_only_no_thermal)
> > +        self.conditions.append(self.sixs_errors)
>
> Should I see some other change for sixs_errors elsewhere in the code?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/USGS-EROS/espa/pull/319/files#r35690135>.
>



-- 
David V. Hill
LSRD
Information Dynamics
Contractor to USGS Earth Resources Observation And Science Center (EROS)
47914 252nd Street
Sioux Falls, SD 57198-9801
605.594.2639
dhill@usgs.gov

Comment (rdilley-usgs) [inline web/ordering/errors.py]: ok

**Pull request merged**